tests/js/spec/views/dashboardsV2/detail.spec.jsx

```diff
@@ -495,6 +495,8 @@ describe('Dashboards > Detail', function () {
       expect(breadcrumbs.exists()).toBe(true);
       expect(breadcrumbs.find('BreadcrumbLink').find('a').text()).toEqual('Dashboards');
       expect(breadcrumbs.find('BreadcrumbItem').last().text()).toEqual('Custom Errors');
+
+      wrapper.unmount();
```

Comment:
Can we add this to the `afterEach` :eyes: Now that I'm seeing it, I think it might be causing an issue in my PR and I hope it could fix it hehehe

Reply:
86b8ec5beaf78db0e3ceda9c2774d4c5a7e00632